FrontEnd/src/views/EditAanganwadi.vue

```diff
@@ -0,0 +1,138 @@
+<template>
+    <div class="container-md mt-5 p-3">
+      <div id="x-contest" class="container-float">
+        <div class="row">
+          <h3 class="float-start">Edit Aanganwadi</h3>
+          <form @submit.prevent="updateAanganwadi">
+            <div class="col-12 my-2">
+              <label for="exampleFormControlInput1">Aanganwadi Contact Person</label>
+              <input
+                type="text"
+                class="form-control"
+                id="exampleFormControlInput1"
+                placeholder="Contact Person"
+                v-model="currentAanganwadi.aanganwadiName"
+              />
+            </div>
+  
+            <div class="col-6 my-2">
+              <label for="exampleFormControlInput1">Contact Phone</label>
+              <input
+                type="text"
+                class="form-control"
+                id="exampleFormControlInput1"
+                placeholder="Phone Number"
+                v-model="currentAanganwadi.contactPersonPhone"
+              />
+            </div>
+  
+            <div class="col-6 my-2">
+              <label for="exampleFormControlInput1">Password</label>
+              <input
+                type="password"
+                class="form-control"
+                id="exampleFormControlInput1"
+                placeholder="Password"
+                v-model="currentAanganwadi.contactPersonPassword"
+              />
+            </div>
+  
+            <div class="col-6 my-2">
+              <label for="exampleFormControlInput1">Location</label>
+              <input
+                type="text"
+                class="form-control"
+                id="exampleFormControlInput1"
+                placeholder="Aanganwadi Location"
+                v-model="currentAanganwadi.location"
+              />
+            </div>
+  
+            <div class="col-6 my-2">
+              <label for="exampleFormControlInput1"
+                >Latitude, Longitude Coordinates</label
+              >
+              <input
+                type="text"
+                class="form-control"
+                id="exampleFormControlInput1"
+                placeholder="Latitude, Longitude Coordinates"
+                v-model="currentAanganwadi.coordinates"
+              />
+            </div>
+            
+            
+            <div class="row">
+              <div class="col-12 p-2">
+                <button
+                  type="submit"
+                  class="btn btn-primary float-end mx-2"
+                  data-nav="admin.ngos"
```

Comment:
rename all ngos to aaganwadis

Reply:
data-nav value renamed to admin.aanganwadis